src/io/address.hpp

```diff
@@ -59,7 +59,7 @@ struct Address {
 
   static Address TestAddress(uint16_t port) {
     return Address{
-        .unique_id = boost::uuids::uuid{boost::uuids::random_generator()()},
+        .unique_id = boost::uuids::uuid{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, static_cast<unsigned char>(port)},
```

Comment:
Please put an MG_ASSERT to make sure `port` is never bigger than 255, because that would clash with lower port numbers. Or maybe populate the last to bytes with port.

Reply:
applied